packages/opentelemetry/src/WebSdk.ts

```diff
@@ -66,9 +66,9 @@ export const layerTracerProvider = (
  */
 export const layer: {
   (evaluate: LazyArg<Configuration>): Layer.Layer<Resource.Resource>
-  <R, E>(evaluate: Effect.Effect<R, E, Configuration>): Layer.Layer<Resource.Resource, E, R>
+  <R, E>(evaluate: Effect.Effect<Configuration, E, R>): Layer.Layer<Resource.Resource, E, R>
```

Comment:
nitpick but, but in an AER world, ER would be the correct order

Reply:
Nice catch, fixed.